mne_bids/tsv_handler.py

```diff
@@ -49,18 +49,24 @@ def _contains_row(data, row_data):
     ----------
     data : collections.OrderedDict
         OrderedDict to check.
-    row_data : list
-        List of values to be searched for. This must match the contents of a
-        row exactly for a positive match to be returned.
+    row_data : dict
```

Comment:
why not an ordered dict?

Reply:
It won't actually change anything. Unless we also test to make sure that the columns are in the same order, but I think that will needlessly add unnecessary complexity.

Reply:
actually, the order of rows is important for the BIDS-validator --> see: https://github.com/bids-standard/bids-validator/blob/6498800e1523ce6dcbe4ac998990a65dc6336aee/validators/tsv/tsv.js#L218-L246

Reply:
yes, but we aren't adding them here, we are just checking to see if the row data is in the given data.
We are already generating everything as ordered dicts, and we will not modify them here, so it is completely safe to just check using an un-ordered dictionary.
I can change it if you really want and make it check that the order is correct, but it feels unnecessary.